query-node/mappings/src/common.ts

```diff
@@ -88,18 +95,20 @@ export function inconsistentState(extraInfo: string, data?: unknown): never {
   // log error
   logger.error(errorMessage, data)
 
+  // eslint-disable-next-line local-rules/no-throw
   throw errorMessage
 }
 
-/*
-  Reports that insurmountable unexpected data has been encountered and throws an exception.
-*/
-export function unexpectedData(extraInfo: string, data?: unknown): never {
-  const errorMessage = 'Unexpected data: ' + extraInfo
+/**
+ * Reports an unimplemented mapping/variant error for which a runtime implementation logic exists and is not filtered.
+ */
+export function unimplementedError(extraInfo: string, data?: unknown): never {
+  const errorMessage = 'unimplemented error: ' + extraInfo
 
   // log error
   logger.error(errorMessage, data)
 
+  // eslint-disable-next-line local-rules/no-throw
   throw errorMessage
```

Comment:
Would it make sense to call `process.exit` here instead of throwing?
This would be to prevent any code capturing this throw, and also caller doesn't need to `return` from the calling function.

Reply:
Addressed in [810240d](https://github.com/Joystream/joystream/pull/4856/commits/810240d2ae555d114778650db8b76823769a603e)